src/App.css

```diff
@@ -0,0 +1,147 @@
+/**************/
+/* App styles */
+/**************/
```

Comment:
Admittedly, in the styles lecture (PR #422 in textbook-curriculum), I mention that every component has its own .css file lololol... we should either acknowledge this in this project's readme that we have it all in one file (maybe even acknowledge that they can change this CSS), or we should change the styles lecture (I'm happy to do that)